CIP-?/README.md

```diff
@@ -0,0 +1,99 @@
+---
+CIP: [To be assigned]
```

Comment:
```suggestion
CIP: ?
```
Specific format required by CIP-0001 and the site generators that parse CIP text.